### PR TITLE
HLT update including Ecal for 74X

### PR DESCRIPTION
**Pull request by Martin-Grunewald**
HLT update including Ecal for 74X: includes #7771 and #7877 and required HLT changes.

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by Dr15Jones** [comment]
+1

**Comment by Martin-Grunewald** [comment]
+1
Rebased to 740pre8

**Comment by Martin-Grunewald** [comment]
please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/7915	967eac3

signoff is based on observations in #7916 and check that the introduced changes in the code are identical in 74X here (comparing to 740pre8) and in 75X (comparing there to CMSSW_7_5_X_2015-02-25-1400)

**Comment by davidlange6** [comment]
+1

**Comment by Martin-Grunewald** [comment]
@perrotta 
I assume you latest commit was based on an IB rather than pre8.
This means we now get added signature requirements for other PRs
integrated in here...
I try to rebase to get rid of these...

**Comment by Martin-Grunewald** [comment]
+1

**Comment by Martin-Grunewald** [comment]
Rebased to CMSSW_7_4_X_2015-02-27-0200

**Comment by Martin-Grunewald** [comment]
@mulhearn 
L1 signature now required as we moved the stage1 L1 unpacker directly into the 25ns HLT menus,
thus removed the customisation of hlt modules for this case from L1 postLS1.

**Comment by Martin-Grunewald** [comment]
please test

File: L1Trigger/L1TCommon/python/customsPostLS1.py [hunk 1]
```diff
@@ -132,6 +97,5 @@ def customiseSimL1EmulatorForPostLS1_HI(process):
     # set the Stage 1 heavy ions-specific parameters
     process.simCaloStage1Digis.FirmwareVersion = cms.uint32(1)
     # move to the heavy ions draft L1 menu once the HLT has been updated accordingly
-    #process = L1Menu_CollisionsHeavyIons2015_v0(process)
-    process = L1Menu_Collisions2015_25ns_v2(process)
+    process = L1Menu_CollisionsHeavyIons2015_v0(process)
```
Comment (fwyzard) [inline L1Trigger/L1TCommon/python/customsPostLS1.py]: @mark-grimes how should we propagate this into Eras ?

Comment (mark-grimes) [inline L1Trigger/L1TCommon/python/customsPostLS1.py]: I'll check over these changes and put in a separate pull request to do the same in Eras.  Thanks for pointing this out.

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/7915	d3ee4a3
there are no additional changes in RECO code since the last signature  from RECO https://github.com/cms-sw/cmssw/pull/7915	967eac3

Notably, there are no changes in DQM in HLT in jenkins tests. Probably all changes are in the workflows that don't run or in paths that are not monitored.

**Comment by Martin-Grunewald** [comment]
@mulhearn 
PING - Michael, waiting for your signature!

**Comment by fwyzard** [comment]
@davidlange6 can you please bypass the L1 signature ?
It is required "only" for changes
  - to the HLT menu
  - to complete my previous #7579

.A

**Comment by mulhearn** [comment]
+1

**Comment by perrotta** [comment]
Thank you @mulhearn !
@davidlange6 : since it is fully signed now, could you please allow this to be merged in 74X?
We have other HLT developments in the queue, and it would be much easier if we don't have to reset all other signatures every time we want to submit them. Thank you, A.

**Comment by davidlange6** [comment]
@perrotta - we should see that things are all fine in 75x first.

Meanwhile, perhaps a new PR is the best way to avoid reseting signatures on this one. (with the added bonus that the title of the PR can be descriptive of the new changes)

**Comment by Martin-Grunewald** [comment]
@davidlange6
What is holding up integration of this PR?

File: L1Trigger/L1TCommon/python/customsPostLS1.py (hunk 1 above)
Comment (Martin-Grunewald) [inline L1Trigger/L1TCommon/python/customsPostLS1.py]: @mark-grimes 
you commented:
```
I'll check over these changes and put in a separate pull request to do the same in Eras. Thanks for pointing this out.
```
What is the status on this - and your resulting PR ?

**Comment by davidlange6** [comment]
+1

File: L1Trigger/L1TCommon/python/customsPostLS1.py (hunk 1 above)
Comment (mark-grimes) [inline L1Trigger/L1TCommon/python/customsPostLS1.py]: Sorry for the delay - #8205.  I think that's all that's required but I'm checking it over as part of a general equivalency check.

**Pull request merged**